kedro/template/{{ cookiecutter.repo_name }}/kedro_cli.py

```diff
@@ -121,8 +121,13 @@ def test(args):
 
 
 @cli.command()
+@click.option("--conda")
 def install():
```

Comment:
This needs to take the flag as argument in order for the call to work.
```suggestion
 def install(conda):
```